pkg/schedule/operator/kind.go

```diff
@@ -49,6 +49,8 @@ const (
 	OpWitnessLeader
 	// Include witness transfer.
 	OpWitness
+	// Include keyrange balancer
+	OpKeyrange
```

Comment:
Can we reuse the "OpRange" instead of adding "OpKeyrange"?

Reply:
This is for scatter range scheduler. IMO, different schedulers do not interfere with each other.